Modules/Filtering/LabelMap/include/itkShapeLabelMapFilter.hxx

```diff
@@ -847,7 +847,7 @@ ShapeLabelMapFilter<TImage, TLabelImage>::ComputeOrientedBoundingBox(LabelObject
   // iterate over all corners (2^D) of the pixel
   for (unsigned int p = 0; p < 1u << ImageDimension; ++p)
   {
-    Vector<double, ImageDimension> spacingAxis(0.5 * spacing);
+    Vector<double, ImageDimension> spacingAxis(spacing * 0.5);
```

Comment:
Reversing these changes something?

Reply:
Yes, this operator reversal fixes `C:\Dev\ITK-git\Modules\Filtering\LabelMap\include\itkShapeLabelMapFilter.hxx(850,52): error C2677: binary '*': no global operator found which takes type 'const itk::Vector<float,3>' (or there is no acceptable conversion)`

Reply:
Egad! Is this due to operator overloading?

Reply:
Of course, what else could it be due? I spent something like 1.5-2 hours on this PR. If you have some will to hunt down the missing/wrong operator overload, that would be good. Some operator is probably having `double` argument instead of `SpacePrecisionType`, or possibly missing a `float` variant.

Reply:
Sometimes C++ is so not obvious. :)

I'm not objecting to the change!, I'm just kinda stunned that such a change could have such an effect, especially when one of the arguments is a literal scalar double.